MatrixSDK/Room/Polls/PollAggregator.swift

```diff
@@ -38,12 +38,16 @@ public class PollAggregator {
     
     private let session: MXSession
     private let room: MXRoom
-    private let pollStartEvent: MXEvent
-    private let pollStartEventContent: MXEventContentPollStart
+    private let pollStartEventId: String
     private let pollBuilder: PollBuilder
     
-    private var eventListener: Any!
+    private var pollStartEventContent: MXEventContentPollStart!
+    
+    private var referenceEventsListener: Any!
+    private var editEventsListener: Any!
```

Comment:
As `referenceEventsListener` is set in init, can it be `: Any`?
Side note: I'd also wonder if `editEventsListener` could actually `Any?` but presumably as `eventListener` was force unwrapped there's more to this elsewhere that is unchanged in this PR.

Reply:
`referenceEventsListener` isn't set in init but what I can do is make them both optional and check in deinit, which would also take care of the `as Any` cast. I honestly can't remember why `referenceEventsListener` was forced unwrapped.